tests/adaptive_time_step.1Rank.sh

```diff
@@ -22,11 +22,14 @@ HIPACE_SOURCE_DIR=$2
 HIPACE_EXAMPLE_DIR=${HIPACE_SOURCE_DIR}/examples/beam_in_vacuum
 HIPACE_TEST_DIR=${HIPACE_SOURCE_DIR}/tests
 
+FILE_NAME=`basename "$0"`
+TEST_NAME="${FILE_NAME%.*}"
+
 # Relative tolerance for checksum tests depends on the platform
 RTOL=1e-12 && [[ "$HIPACE_EXECUTABLE" == *"hipace"*".CUDA."* ]] && RTOL=1e-5
 
-rm -rf negative_gradient_data
-rm -rf positive_gradient_data
+rm -rf negative_gradient.txt
+rm -rf positive_gradient.txt
```

Comment:
Could you comment on these changes?

Reply:
I noticed that the adaptive time step CI test is somewhat outdated. It did not use the test name as output etc. I copy pasted the radiation reaction CI test from this and then noticed that I used the data of the wrong one, because it was not named correctly. Thereby, I noticed that this test does not use any output, but the text output by the code, so these lines fix the cleaning of previous data (there is not `negative_gradient_data` but only `negative_gradient.txt`).
Therefore also the changes below to set this test up to date